__mocks__/electron.js

```diff
@@ -40,6 +40,8 @@ class MockApp extends events.EventEmitter {
       () => os.tmpdir()
     );
     this.setPath = jest.fn();
+    this.getName = jest.fn();
+    this.getVersion = jest.fn();
```

Comment:
Are these needed because of the switch to `jsdom` environment?

Reply:
Hmm, I can't quite recall why these are needed now and they weren't before. I do remember adding these because a test was triggering these API calls and they're not available outside an electron runtime. So these mocks are necessary, I just can't explain why we didn't need them here earlier.